### PR TITLE
Accept path without a starting /

### LINKED ISSUE #8: Accept the endpoint without starting with /
It happens to me from now and then, so I figure I dropped this feedback in case I'm not the only one:

Sometimes, I do this and surprised by the error message:

```
› h api get apps
undefined:1
400 Bad Request
    ^

SyntaxError: Unexpected token B in JSON at position 4
    at JSON.parse (<anonymous>)
    at Request.parseBody (/Users/rbarroso/.local/share/heroku/plugins/node_modules/heroku-api-plugin/node_modules/heroku-client/lib/request.js:256:17)
    at Request.handleFailure (/Users/rbarroso/.local/share/heroku/plugins/node_modules/heroku-api-plugin/node_modules/heroku-client/lib/request.js:286:19)
    at /Users/rbarroso/.local/share/heroku/plugins/node_modules/heroku-api-plugin/node_modules/heroku-client/lib/request.js:163:14
    at IncomingMessage.<anonymous> (/Users/rbarroso/.local/share/heroku/plugins/node_modules/heroku-api-plugin/node_modules/heroku-client/lib/request.js:392:5)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:975:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

Then I realize that I forgot including the `/` at the beginning.

```
› h api get /apps
... # my apps
```

I think it'd be nice to make it work without the first `/`.

### PR DESCRIPTION
**Pull request by raulb**
Implement #8

### CODE REVIEW

File: test/commands/api.js [hunk 1]
```diff
@@ -17,9 +18,8 @@ describe('api', function () {
   it.skip('displays the app info', function () {
```
Comment (dickeyxxx) [inline test/commands/api.js]: should this be skipped?

Comment (raulb) [inline test/commands/api.js]: I don't know. it was already there

Comment (raulb) [inline test/commands/api.js]: @dickeyxxx https://github.com/heroku/heroku-api-plugin/commit/1192ed46a47e71af04e60355a613abd2fa8cb1c2

Comment (dickeyxxx) [inline test/commands/api.js]: yeah but you made changes to this test

**Comment by dickeyxxx** [comment]
I added this in when converting to v6

**Pull request closed without merging**